src/App.css

```diff
@@ -16,9 +16,269 @@ h1 {
   text-align: center;
   font-family: 'buffiedregular';
   font-size: 80px;
+  text-shadow:
+       3px 3px 8px #af000e, 
+       3px 3px 8px #000, 
+       -1px -1px 0 #000, 
+       -1px 2px 2px #fff, 
+       2px 2px 2px #000
 }
```

Comment:
I like the text effects on the header! I think we could up the font size a bit to make it stand out more.